nitrates/lib/calc_BAT_ul.py

```diff
@@ -0,0 +1,286 @@
+import numpy as np
+from astropy.table import Table, vstack
+import matplotlib.pyplot as plt
+import os
+
+#cwd = os.getcwd()
+
+#drm_dir_old = cwd+'/rsp_maskweight/'
+#drm_dir_nitrates = cwd+'/rsp_NITRATES/'
+drm_dir_old = '/gpfs/group/jak51/default/responses/rsp_maskweight/'
+drm_dir_nitrates = '/gpfs/group/jak51/default/responses/rsp_NITRATES/'
```

Comment:
this should use the directory defined in config.py, not hardcoded here

Reply:
the resp dir I have also included as a cli . I have also added it to the config.py file.